# --- Request ---
string robot_name
geometry_msgs/Point end_point
---
# --- Response ---
nav_msgs/Path path
bool success
string message